src/ontology/mondo-ingest.Makefile

```diff
@@ -275,3 +274,30 @@ mappings/mondo-sources-all-lexical.sssom.tsv: $(SCRIPTSDIR)/match-mondo-sources-
 	# The $^ includes the python script, basically all paramters after the colon :
 
 lexical_matches: mappings/mondo-sources-all-lexical.sssom.tsv
+
+#############################
+###### Slurp pipeline #######
+#############################
+slurp/:
+	mkdir -p $@
+
+# TODO: Move this to Makefile. how, when I can't edit it here?
+.PHONY: component-download-mondo.owl
```

Comment:
@Nico: What is it that makes this target `.PHONY`? I thought that was supposed to mean it was a command that did not produce a file or dir with of the target name. But in this case, it does. Is it `.PHONY` because it only shows the filename, but not the path?

Reply:
Btw, the reason I added the goals `component-download-mondo.owl` and `$(COMPONENTSDIR)/mondo.owl` is because it looks like they were required in order to resolve the dependency `$(REPORTDIR)/mirror_signature-mondo.tsv`.

Reply:
This was a silly design choice by my past self, sorry about that. Changing it now will cause too much churn. 

I may change this make pipeline a tiny bit during our next call, but for now, if it works, it looks ok.

Reply:
Ok, that all sounds fine to me! I'll leave it up to you when to change it too.